### DISCUSSION TITLE
Start with views hidden?

### DISCUSSION THREAD

**Discussion by zachlim98** [Textualize/textual#134] [General]
I've been messing about with textual and trying to get two sidebars (left and right). However, it cramps up the middle text when the app first runs because the default for `action_toggle` is to start it with widget.visible. Is there anyway to set a view as False on app load?

**Comment by willmcgugan** [comment]
When you create the view, you can do `my_view.visible = False`

**Reply by zachlim98** [reply]
Thanks for your reply Will! That's what I thought too but I couldn't figure out where to set that attribute. For instance, if I were to create a right side bar with 
`self.view.dock(self.rsidebar, edge="right", size=30, name="rsidebar")` where would I set the view visibility? This is assuming `self.rsidebar = WindowView(Panel("This is the sidebar at the right"))`

**Comment by willmcgugan** [comment]
Set the visibility on rsidebar before you dock it...

```python
self.rsidebar.visible = False
```

**Reply by tusharsadhwani** [reply]
I suppose `visible` can be added as a parameter to the constructor if it doesn't exist already?

**Reply by zachlim98** [reply]
I see - thanks so much for your help Will. Am loving this project and am excited to support it.
